plugins/inputs/t128_metrics/t128_metrics.go

```diff
@@ -173,6 +182,12 @@ func (plugin *T128Metrics) retrieveMetrics(index int, acc telegraf.Accumulator,
 			message = []byte("")
 		}
 
+		if strings.Contains(string(message), "No configured endpoints satisfy the request") {
```

Comment:
This works for individual retrieval. But for bulk retrieval, there could be a bunch of metrics in a given request and they may not all be missing. I think we'll need to parse out the specific IDs and remove them from the retriever itself. Otherwise we might skip a bunch of metrics we don't intend to.